app/api/routers/breeze_buddy/credentials/__init__.py

```
@@ -0,0 +1,152 @@
+            raise HTTPException(
+                status_code=status.HTTP_403_FORBIDDEN,
+                detail=f"Access denied to merchant {merchant_id}",
+            )
+
+    return await list_credentials_handler(merchant_id, current_user)
+
+
+@router.get("/credentials/{credential_id}", response_model=Credential)
+async def get_credential_endpoint(
+    credential_id: str,
+    current_user: UserInfo = Depends(get_current_user_with_rbac),
```

Ruff (0.15.1)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)